app/src/main/java/com/quran/labs/androidquran/ui/QuranActivity.kt

```diff
@@ -55,6 +55,20 @@ import java.util.concurrent.TimeUnit.MILLISECONDS
 import javax.inject.Inject
 import kotlin.math.abs
 
+/**
+ * The home screen activity for the app. Displays a toolbar and 3 fragments:
+ *
+ *  * [SuraListFragment]
```

Comment:
Why not link them too?

Reply:
Those are links https://kotlinlang.org/docs/reference/kotlin-doc.html#linking-to-elements